libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -214,6 +215,20 @@ def _func(
         config_list = get_config_list(config, len(tool_calls))
         with get_executor_for_config(config) as executor:
             outputs = [*executor.map(self._run_one, tool_calls, config_list)]
+
+        commands: list[Command] = [
+            output for output in outputs if isinstance(output, Command)
+        ]
+        if len(commands) > 0:
```

Comment:
why is this? you can just wrap non-command outputs with command

Reply:
was also contemplating wrapping, will just do that, cause technically nothing wrong with it. i just can't imagine a use case where mixing them is valuable but i am sure someone would find it